app/helpers/wishlists_helper.rb

```diff
@@ -0,0 +1,2 @@
+module WishlistsHelper
```

Comment:
I'd remove all empty files. Add them to the repo, when they've got body and are used.